Bullet-dodging game/Bullet-Dodging Game in Processing.md

```diff
@@ -0,0 +1,2191 @@
+# Bullet-Dodging Game in Processing
+
+Processing is an open-source graphical library that uses the Java language. Designed for the visual arts community, Processing can be used to create interactive graphics such as data visualizations, digital art, and video games. In this blog, we're going to build a simple bullet-dodging game from scratch in Processing. All of the built-in functions mentioned below can be found on the [Processing reference page](https://processing.org/reference/).
+
+
+
+## Table of contents:
+
+* [Part I: Getting Started](#part-i:-getting-started)
+* [Part II: Adding the Player and the Bullets](#part-ii:-adding-the-player-and-the-bullets)
+
+* [Part III: Adding a Life and Scoring System](#part-iii:-adding-a-life-and-scoring-system)
+
+* [Part IV: Adding Enemies and Shooting](#part-iv:-adding-enemies-and-shooting)
+
+* [Part V: Implementing More Subclasses](#part-v:-implementing-more-subclasses)
+
+* [Part VI: Adding Images](#part-vi:-adding-images)
+
+* [Part VII: Adding Obstacles](#part-vii:-adding-obstacles)
+
+* [Part VIII: Pausing and Restarting the Game](#part-viii:-pausing-and-restarting-the-game)
+
+
+
+## Part I: Getting Started

```

Comment:
consider adding a way to preview the app you're going to build throughout the tutorial, would make the descriptions in this section and on easier to visualize. just a link to a running version should suffice

Reply:
Yeah. I would be nice if I could directly show the program in the blog, but looks like those online editors that I could find are not compatible with Processing. I have inserted a few more images to better visualize the blog. Do you have any other alternatives?